tests/e2e/util.go

```diff
@@ -1884,7 +1884,7 @@ func getPersistentVolumeSpecWithStorageclass(volumeHandle string, persistentVolu
 }
 
 // getPVCSpecWithPVandStorageClass is to create PVC spec with given PV , storage class and label details
-func getPVCSpecWithPVandStorageClass(pvcName string, namespace string, labels map[string]string, pvName string, storageclass string) *v1.PersistentVolumeClaim {
+func getPVCSpecWithPVandStorageClass(pvcName string, namespace string, labels map[string]string, pvName string, storageclass string, diskSize string) *v1.PersistentVolumeClaim {
```

Comment:
this declaration of `diskSize` can be easily confused with `diskSize` from e2e _common.go

Reply:
renamed to sizeOfDisk